_compdemos/EconomyCommand-API.md

```diff
@@ -307,11 +308,11 @@ instead of using the swift command from the unix shell one can also access swift
 
 
 
-​## access from the command line or Rest API
+## access from the command line or Rest API
```

Comment:
This section is rather unclear to me- it seems that there may be some missing information about what those access methods are or what "direct access" is.  I think we also need a link for the reference to"How to Access the Swiftstack Object Store"